samples/embedded-auth-with-sdk/src/main/java/com/okta/spring/example/controllers/LoginController.java

```diff
@@ -136,6 +144,51 @@ public ModelAndView selectAuthenticator(final @RequestParam("authenticator-type"
 
         ProceedContext proceedContext = Util.getProceedContextFromSession(session);
 
+        if (authenticatorType != null && authenticatorType.equals("Security Key or Biometric")) {
+            ModelAndView enrollModelAndView = new ModelAndView("enroll-webauthn-authenticator");
```

Comment:
What is ModelAndView? why do you need to pass a remediation option name? Do devs have to know this to use the feature?